packages/2018-neighborhood-development/src/state/voters-on-the-move/selectors.js

```diff
@@ -0,0 +1,37 @@
+import { createSelector } from 'reselect';
+import { rootState } from '../selectors';
+
+export const getVotersOnTheMoveRequest = createSelector(
+  rootState,
+  ({ votersOnTheMove }) => votersOnTheMove,
+);
+
+export const getVotersOnTheMoveData = createSelector(
```

Comment:
I'm sure it's a pattern we're using throughout the app, but it seems like overkill to use selectors here. Since we're not computing any derived data from state, I don't think there's any benefit and it makes it a little more confusing than just doing something like this in our `mapStateToProps` functions:

```
export default connect(
  state => ({
    votersOnTheMove: state.neighborhood.votersOnTheMove,
    ...
  }),
);
```

I know we're composing them with `getVotersOnTheMoveRequest ` to dry up the code but that's not computing anything either. I'm sure I'm missing something, though, so just curious!

Reply:
I'll bet @DingoEatingFuzz has some thoughts on this pattern, which I'd be interested to hear as well!

Reply:
I do have thoughts! You are correct, @nrbernard,  that `getVotersOnTheMoveRequest` itself isn't doing much, just peeling a property off of the response, but it is built on top of the `rootState` selector that conditionally uses the true root of state or the scope project root of state (i.e., `root.neighborhoodDevelopment`). 

By skipping selectors and doing the quick property lookup in the component as you suggest, you have to increase the surface area where we have to consider the store merging that happens when running the 2018 package vs. the neighborhood-development package in isolation.

All that said, I'm not opposed to rethinking our approach to this, but I do think it's nice that the store merging thing is abstracted to the point where it can essentially go unnoticed.

More on this in [ARCHITECTURE.md](https://github.com/hackoregon/civic/blob/master/ARCHITECTURE.md#creating-a-unified-redux-store).